apparmor.d/profiles-s-z/udisksd

```diff
@@ -54,6 +54,14 @@ profile udisksd @{exec_path} flags=(attach_disconnected) {
   umount @{run}/udisks2/temp-mount-*/,
   umount /media/cdrom[0-9]/,
 
+  dbus (send,receive) bus=system path=/
+       interface=org.freedesktop.DBus.Introspectable
+       member=Introspect,
+
+  dbus (send,receive) bus=system path=/
+       interface=org.freedesktop.DBus.Properties
+       member=Get,
```

Comment:
Was there any peer? Or there were too many?

Reply:
I believe this one was `label_name=Unconfined`.

Reply:
Alright, that's a subject for refactoring. I think it should be left as is.